openpype/hosts/max/plugins/publish/validate_loaded_plugin.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Validator for USD plugin."""
+from pyblish.api import InstancePlugin, ValidatorOrder
```

Comment:
@iLLiCiTiT is this a case where this Plugin will now also get picked up by Pyblish because it finds the plug-in inside this file? Or does it skip the base classes?

Reply:
I think that pyblish implementation of plugins discovery does not skip them. @moonyuet please use `import pyblish.api` and `pyblish.api.InstancePlugin` for class source.